test/clients/openshift/template.go

```diff
@@ -23,7 +23,7 @@ func init() {
 	// additional OpenShift types must be registered manually.  AddToScheme
 	// registers the API group types (e.g. route.openshift.io/v1, Route) only.
 
-	if err := templatev1.AddToSchemeInCoreGroup(scheme.Scheme); err != nil {
+	if err := templatev1.Install(scheme.Scheme); err != nil {
```

Comment:
this is causing the test fail, busy investigating..

Reply:
Any follow-up on this?

Reply:
I fixed it here: https://github.com/openshift/openshift-azure/pull/1890/files#diff-aaf8e1ec8557614866fe4f35fa93a036